### PR TITLE
feat: add new member storage items type

### PR DESCRIPTION
**Pull request by mariembencheikh**
#588

### CODE REVIEW

File: src/member/member.ts [hunk 1]
```diff
@@ -9,6 +9,30 @@ export type MemberStorage = {
...
+  parent?: {
+    id: UUID;
+    name: string;
+  };
+};
+
+export type Pagination = {
+  totalItems: number;
+  totalPages: number;
+  page: number;
```
Comment (Defferrard) [inline src/member/member.ts:27]: From the specification, should it be `currentPage` ?
```suggestion
  currentPage: number;
```

**Review by Defferrard — COMMENTED** [review]
LGTM, good job! I just saw an incoherence with the specification, not sure if it is intended

**Review by ReidyT — APPROVED** [review]
Thanks for this PR, it looks good to me 😆 I just agree with @Defferrard concerning the difference with the specification.

File: src/member/member.ts (hunk 1 above)
Comment (mariembencheikh) [reply src/member/member.ts:27]: Yes, you're right

**Review by Defferrard — APPROVED** [review]
LGTM!

**Pull request merged**